core/tasks.py

```diff
@@ -238,3 +243,24 @@ def _overall_score_and_result_type(datafile):
     result_type = ScoreResult.ResultTypes.SIMPLE
     overall_score = float(datafile.readline())
     return overall_score, result_type
+
+
+def _get_mimetype(file_path):
+    mimetype = magic.from_file(file_path, mime=True)
+    # check if mimetype has something useful
+    # empty file returns inode/x-empty
+    if mimetype == 'inode/x-empty':
+        # if no mime type, it can be null(None)
+        return None
+    unzipped_file = magic.Magic(mime=True, uncompress=True)
+    unzipped_file_mimetype = unzipped_file.from_file(file_path)
+    zipped_file = magic.Magic(mime=True, uncompress=False)
+    zipped_file_mimetype = zipped_file.from_file(file_path)
+    # if the file is unzipped
+    if unzipped_file_mimetype == zipped_file_mimetype:
+        return unzipped_file_mimetype
+    else:
+        # seperate unzipped and zipped mimetype by comma
+        file_mimetype = ('%s,%s' % (unzipped_file_mimetype, zipped_file_mimetype))
```

Comment:
There is no need to wrap this in parenthesis (e.g., `file_mimetype = '%s,%s' % (unzipped_file_mimetype, zipped_file_mimetype)` is better).